easylist_adult/adult_specific_hide.txt

```diff
@@ -642,6 +644,7 @@ efukt.com##.media_below_container
 lesbianbliss.com,mywebcamsluts.com,transhero.com##.media_spot
 ryuugames.com##.menu-item > a[href^="https://l.erodatalabs.com/s/"]
 camcam.cc##.menu-item-5880
+gamcore.com##.menuArea a[href="https://www.porngames.tv/"]
```

Comment:
@rakleed This and the rule below are not ads, but links to their partner sites (also linked back by the destination). The linking is one of the fundamentals of the Internet and we don't want to break this.